lib/web_ui/lib/src/engine/canvaskit/color_filter.dart

```diff
@@ -83,10 +83,14 @@ class CkBlendModeColorFilter extends CkColorFilter {
 
   @override
   SkColorFilter _initRawColorFilter() {
-    return canvasKit.ColorFilter.MakeBlend(
+    SkColorFilter filter = canvasKit.ColorFilter.MakeBlend(
```

Comment:
If `MakeBlend` is allowed to return null, then we should change the API binding in `canvaskit_api.dart` to return `SkColorFilter?`.

Reply:
Done